runtime/lua/vim/treesitter.lua

```diff
@@ -147,7 +147,7 @@ end
 ---
 ---@param node_or_range table
 ---
----@returns start_row, start_col, end_row, end_col
+---@returns table start_row, start_col, end_row, end_col
 function M.get_node_range(node_or_range)
```

Comment:
Is this a range or a region? In https://github.com/neovim/neovim/pull/13896 we are discussing vim.region(), which implies a bounding box. Whereas range is 1-dimensional (doesn't have columns).

However, maybe it's too late and "range" simply is understood as "region"?

Reply:
I think "TS range" is used as a term throughout this code base. 

@vigoux ?

Reply:
Hmm, I guess the point is that a `region` can be any shape (including block), while a `range` here is strictly linear even if it may span multiple lines?

Reply:
yes

Reply:
I.e., call everything a range and remove "region" from the code?

Reply:
yes. It's either that or use "region" consistently, imo.

Reply:
Noted. I prefer range then.